apps/frontend/src/app/poll/ical/ical.component.ts

```diff
@@ -115,10 +112,7 @@ export class IcalComponent implements OnInit {
       if (!this.config.emptyEvents && !e.participants) {
         return false;
       }
-      if (this.config.onlyBookedEvents && !this.poll?.bookedEvents[e._id]) {
```

Comment:
Here too. The idea is to have a list of exit conditions and only if all of them were ok we go to the return true